src/devices/thermostat.cpp

```diff
@@ -4912,7 +4912,7 @@ void Thermostat::register_device_values_dhw(std::shared_ptr<Thermostat::DhwCircu
                               DeviceValueUOM::MINUTES,
                               MAKE_CF_CB(set_wwchargeduration));
         register_device_value(tag, &dhw->wwCharge_, DeviceValueType::BOOL, FL_(wwCharge), DeviceValueUOM::NONE, MAKE_CF_CB(set_wwcharge));
-        register_device_value(tag, &dhw->wwExtra_, DeviceValueType::UINT8, FL_(wwExtra), DeviceValueUOM::DEGREES);
+        register_device_value(tag, &dhw->wwExtra_, DeviceValueType::UINT8, FL_(wwExtra), DeviceValueUOM::NONE);
```

Comment:
No complain in all these years ;-) 
But the comment in https://github.com/emsesp/EMS-ESP32/blob/9689f0a7da258d5cc27cb9e8a588f98a736a3141/src/devices/thermostat.cpp#L1249 says it is bool.

Reply:
that's what I thought, but if you scroll up you see it also can have a value of 2. So?? not sure

Reply:
Did not find this. The comment above describes the telegram offsets.
I think it's bool.

Reply:
i think you're right. will change